### DISCUSSION TITLE
Account was flagged, contacted support, been waiting 2 months for a response

### DISCUSSION THREAD

**Discussion by xavwasonios** [community/community#134060] [Other Feature Feedback, Questions, & Ideas] [answered]
A few months ago, a few of my accounts were compromised. This unfortunately includes my GitHub account (@xavwashere.) During this period, my GitHub account was flagged. Unfortunately, I didn't notice this for 2 months, until I made a support ticket in hopes of my account getting unflagged. That was 2 months ago (May 28.) If anyone knows what I can do, please let me know.
(to any GitHub staff, ny support ticket is #2815618)

Thanks,
-xavvvv

**Comment by queenofcorgis** [comment, marked as answer]
Hi @xavwasonios 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
